source/features/bugs-tab.tsx

```diff
@@ -84,7 +84,6 @@ async function init(): Promise<void | false> {
 	// Set temporary counter
 	const bugsCounter = select('.Counter', bugsTab)!;
 	bugsCounter.textContent = '0';
-	bugsCounter.title = '';
```

Comment:
We still need to reset this as soon as possible because the count is loaded later